modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc

```diff
@@ -108,20 +115,18 @@ DiffusionMIPSolver::AssembleAand_b_wQpoints(const std::vector<double>& q_vector)
             entry_aij +=
               sigr_g * qp_data.ShapeValue(i, qp) * qp_data.ShapeValue(j, qp) * qp_data.JxW(qp);
 
-            if (source_function.empty())
+            if (!source_function_)
```

Comment:
Convention... Most everywhere we have tried to use `not` `and` `or`. We should stick to that if possible.

Reply:
I was not aware you had a convention for this and that you prefer the spelled out version. I will fix that in this PR and I will create a task for me to review the code base for `&&`, `||` and `!` and change those instances if I find any.

Reply:
Fixed